CLAUDE.md

```
@@ -204,6 +204,25 @@ Note: there is **no query cache**. The only caching middleware ever present —
 reads and mutations under one name, making per-subaction cache exclusion impossible. The
 `health/diagnose` "caching disabled" note is simply accurate.
 
+### HTTP Authentication (bearer token OR Google OAuth)
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)